en/release_note/release_note_latest_ver.rst

```diff
@@ -22,62 +24,66 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
 
-The database volume of CUBRID 10.2 is not compatible with that of CUBRID 10.1 and earlier versions. Therefore, if you use CUBRID 10.1 or earlier, you must **migrate your databases**. Regarding this, see :doc:`/upgrade`.
+The database volume of CUBRID 11.0 is not compatible with that of CUBRID 10.2 and earlier versions. Therefore, if you use CUBRID 10.1 or earlier, you must **migrate your databases**. Regarding this, see :doc:`/upgrade`.
 
 .. TODO: coming soon 
 
 Driver Compatibility
 --------------------
 
-*   The JDBC and CCI driver of CUBRID 10.2 are compatible with the DB server of CUBRID 10.1, 10.0, 9.3, 9.2, 9.1, 2008 R4.4, R4.3 or R4.1.
+*   The JDBC and CCI driver of CUBRID 11.0 are compatible with the DB server of CUBRID 10.2, 10.1, 10.0, 9.3, 9.2, 9.1, 2008 R4.4, R4.3 or R4.1.
 *   To upgrade drivers are highly recommended.
 
-We strongly recommend to also upgrade your drivers to use CUBRID 10.2. Some new features, especially for JSON data types are only supported with 10.2 drivers.
+As new features such as result cache have been improved in the CUBRID 11.0 driver, CUBRID 11.0 users are strongly recommended to upgrade the driver. 
 
-For more details on changes, see the :ref:`10_2_changes`. Users of previous versions should check the :ref:`10_2_changes` and :ref:`10_2_new_cautions` sections.
+For more details on changes, see the :ref:`11_0_changes`. Users of previous versions should check the :ref:`11_0_changes` and :ref:`11_0_new_cautions` sections.
 
 .. _11_0_changes:
 
-10.2 Changes
+11.0 Changes
 ============
 
-Please refer to `change logs of CUBRID 10.2 <https://github.com/CUBRID/cubrid/releases/tag/v10.2>`_.
+Please refer to `change logs of CUBRID 11.0 <https://github.com/CUBRID/cubrid/releases/tag/v11.0>`_.
 
 
 Cautions
 =========
 
-.. _10_2_new_cautions:
+.. _11_0_new_cautions:
 
 New Cautions
 ------------
 
-The database volume of CUBRID 10.2 is not compatible with that of CUBRID 10.1 and earlier versions.
+The database volume of CUBRID 11.0 is not compatible with that of CUBRID 10.2 and earlier versions.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When creating a table without an option, it is created as a reuse_oid table.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The maximum length of the CHAR data type has been changed to 256M character string.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Modified to occur error when the length of a string is larger than the length was entered in the string data type.
```

Comment:
@charischau 
how about changing the following ?
Modified to occur error when the input string length is longer than the set length of the string data type.

Reply:
sounds good to me  :)

Reply:
I fixed it.